contact.html

```diff
@@ -30,27 +30,27 @@
 <body>
     <header>
         <nav class="navbar navbar-expand-md navbar-light fixed-top bg-light">
-            <a class="navbar-brand" href="index">
+            <a class="navbar-brand" href="index.html">
                 <img src="assets/brand/logo-csi.jpg" width="40" height="40" class="d-inline-block align-top" alt="CSI Logo"> <span class="logo-title">CSI</span> </a>
                 <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
                     <span class="navbar-toggler-icon"></span>
                 </button>
                 <div class="collapse navbar-collapse" id="navbarCollapse">
                     <ul class="navbar-nav mr-auto">
                         <li class="nav-item">
-                            <a class="nav-link" href="index">Home</a>
+                            <a class="nav-link" href="index.html">Home</a>
```

Comment:
GitHub Pages supports links without extensions. It's fine if we choose to skip it and use a local http/jekyll server instead.